integr8ly/cloud-resource-operator
# MGDAPI-4628 G404: Insecure random number source (rand)

## Overview

Jira: [MGDAPI-4628](https://issues.redhat.com/browse/MGDAPI-4628)

## Verification

- Checkout this pr `gh pr checkout 631`
- Run `make gosec`
- Ensure that there are no issues.
